config/defaultAccountOverride.ts

```diff
@@ -0,0 +1,72 @@
+import findup from 'findup-sync';
+import fs from 'fs-extra';
+
+import { getCwd } from '../lib/path';
+import {
+  DEFAULT_ACCOUNT_OVERRIDE_ERROR_INVALID_ID,
+  DEFAULT_ACCOUNT_OVERRIDE_ERROR_ACCOUNT_NOT_FOUND,
+  DEFAULT_ACCOUNT_OVERRIDE_FILE_NAME,
+} from '../constants/config';
+import { i18n } from '../utils/lang';
+import { FileSystemError } from '../models/FileSystemError';
+import { getAllConfigAccounts } from './index';
```

Comment:
I opted to move this stuff to its own file because it doesn't directly interact with the config file